pkgs/applications/graphics/pbrt/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
 
-  version = "2016-05-19";
+  version = "2017-01-12";
   name = "pbrt-v3-${version}";
   src = fetchgit {
     url = "https://github.com/mmp/pbrt-v3.git";
-    rev = "638249e5cf4596e129695c8df8525d43f11573ff";
-    sha256 = "10ykqrg4zcfb4sfsg3z793c6vld6b6g8bzfyk7ya3yvvc9sdlr5g";
+    rev = "35b6da3429526f2026fe5e5ebaf36d593e113028";
+    sha256 = "10lvbph13p6ilzqb8sgrvn9gg1zmi8wpy3hhjbqp8dnsa4x0mhj7";
   };
 
   fetchSubmodules = true;
```

Comment:
meta = {
     homepage = "http://pbrt.org";
     description = "The renderer described in the third edition of the book 'Physically Based Rendering: From Theory To Implementation'";
     platforms = stdenv.lib.platforms.linux ;
-    license = stdenv.lib.licenses.bsd3;
+    license = stdenv.lib.licenses.bsd2;
     maintainers = [ stdenv.lib.maintainers.juliendehos ];
     priority = 10;
   };

Could do:
meta = stdenv.lib; {
  homepage = "http://pbrt.org";
  description = "The renderer described in the third edition of the book 'Physically Based Rendering: From Theory To Implementation'";
  platforms = platforms.linux;
 license = licenses.bsd2;
 maintainers = [ maintainers.juliendehos ];
 priority = 10;